packages/babel-core/test/get-possible-preset-names.js

```diff
@@ -0,0 +1,22 @@
+let assert                 = require("assert");
+let getPossiblePresetNames = require("../lib/helpers/get-possible-preset-names");
+
+describe("getPossiblePresetNames", function () {
+  it("adds the babel-preset prefix", function() {
+    assert.deepEqual(getPossiblePresetNames("foobar"), ["babel-preset-foobar", "foobar"]);
+  });
+
+  it("inserts babel-preset after @org/", function() {
+    assert.deepEqual(getPossiblePresetNames("@babel/es2015"), [
```

Comment:
I'm kind of thinking if you did `@babel/es2015` you probably would ever mean ` "babel-preset-@babel/es2015",` so we should remove this option?